src/main/resources/META-INF/accesstransformer.cfg

```diff
@@ -1,59 +1,4 @@
-public net.minecraft.block.ConcretePowderBlock field_200294_a # solidifiedState
-public-f net.minecraft.client.renderer.model.Model func_228282_a_(Lnet/minecraft/util/ResourceLocation;)Lnet/minecraft/client/renderer/RenderType; # getRenderType
-public net.minecraft.client.renderer.RenderState field_228509_a_ # name
-public net.minecraft.entity.ai.brain.task.GiveHeroGiftsTask field_220403_a # GIFTS
-public net.minecraft.item.crafting.RecipeManager func_215366_a(Lnet/minecraft/item/crafting/IRecipeType;)Ljava/util/Map; #getRecipes
-public net.minecraft.client.Minecraft field_211501_aD # fontResourceMananger
-public net.minecraft.entity.projectile.ProjectileEntity field_234609_b_ # shootingEntity
-public net.minecraft.loot.functions.LootFunctionManager func_237451_a_(Ljava/lang/String;Lnet/minecraft/loot/ILootSerializer;)Lnet/minecraft/loot/LootFunctionType; #func_237451_a_
-public net.minecraft.loot.LootEntryManager func_237419_a_(Ljava/lang/String;Lnet/minecraft/loot/ILootSerializer;)Lnet/minecraft/loot/LootPoolEntryType; #func_237419_a_
-public net.minecraft.client.gui.fonts.FontResourceManager field_238546_d_ #field_238546_d_
-# Used for FastResourceAccess
-public net.minecraft.resources.SimpleReloadableResourceManager field_199014_c # namespaceResourceManagers
-# Used for ChatUtils
-public net.minecraft.client.gui.NewChatGui func_146242_c(I)V # deleteChatLine
-public net.minecraft.client.gui.NewChatGui func_146234_a(Lnet/minecraft/util/text/ITextComponent;I)V # printChatMessageWithOptionalDeletion
-# Used for world gen in 1.16.2
-public net.minecraft.world.gen.feature.Features func_243968_a(Ljava/lang/String;Lnet/minecraft/world/gen/feature/ConfiguredFeature;)Lnet/minecraft/world/gen/feature/ConfiguredFeature; #register
-public net.minecraft.world.gen.feature.Feature func_214468_a(Ljava/lang/String;Lnet/minecraft/world/gen/feature/Feature;)Lnet/minecraft/world/gen/feature/Feature; #register
-# Used for village generation
-public net.minecraft.world.gen.feature.jigsaw.LegacySingleJigsawPiece <init>(Lcom/mojang/datafixers/util/Either;Ljava/util/function/Supplier;Lnet/minecraft/world/gen/feature/jigsaw/JigsawPattern$PlacementBehaviour;)V
-# Used to mark Fabulous as unsupported when stencil is enabled
-public net.minecraft.client.renderer.GPUWarning field_241688_c_ # warnings map
-public net.minecraft.world.gen.feature.template.Template$Palette <init>(Ljava/util/List;)V # Template.Palette constructor
-public net.minecraft.world.gen.feature.template.Template field_186272_c # size
+# Used in ModelPowerPack, remove final modifier
+public-f net.minecraft.client.renderer.model.Model func_228282_a_(Lnet/minecraft/util/ResourceLocation;)Lnet/minecraft/client/renderer/RenderType; #getRenderType
+# Used in TurnAndCopyRecipe, make protected to allow overrides
+protected net.minecraft.item.crafting.ShapedRecipe func_77573_a(Lnet/minecraft/inventory/CraftingInventory;IIZ)Z #checkMatch
```

Comment:
Is there a reason we are keeping these?

Reply:
Yes, we override both of these, which (as far as I can tell) isn't possible with Mixins. The first one is final, so we can't override it without removing that modifier; the second one is private, so we need to make it (at least) protected.

I just noticed that we can actually get rid of the first one cleanly as that method just applies a `Function` stored in a field (will push that in a minute), removing the second one would require copying large parts of `ShapedRecipe`, so I'm keeping it for now.

Reply:
Actually, thought: we don't really need the override for anything but the "turn" functionality. It might be an idea to move that functionality to datagen (i.e. generate one "unturned" and one "rotate by x degrees" recipe), then we could probably get rid of ATs completely.

Reply:
That would mean listing multiple recipes in JEI. I don't want that.